lib/token.rb

```diff
@@ -0,0 +1,13 @@
+module Token
+    def hero
+        return :hero
```

Comment:
What does hero mean?

Reply:
It's just the token I'm using for each player.

Reply:
It may be clearer if it's just `X` and `O` as these are the default Tic Tac Toe game markers